### PR TITLE
DAC-385 Test History  + DAC-301

### PR DESCRIPTION
**Pull request by amnambiar**
Imp: To be merged only after https://github.com/input-output-hk/dapps-certification/pull/30

See this link https://github.com/input-output-hk/dapps-certification/compare/DAC-317...DAC-385 for proper comparison of changes

### CODE REVIEW

File: react-web/package.json [hunk 1]
```diff
@@ -18,7 +18,9 @@
     "@types/node": "^16.11.45",
     "@types/react": "^18.0.15",
     "@types/react-dom": "^18.0.6",
+    "@types/react-table": "^7.7.12",
     "axios": "^0.27.2",
+    "moment": "^2.29.4",
```
Comment (bogdan-manole) [inline react-web/package.json:23]: https://terodox.tech/migrating-away-from-momentjs-part1/

I would go rather with `date-fns` or any other modern maintained library

File: react-web/src/pages/testHistory/TestHistory.tsx [hunk 2]
```diff
@@ -1,8 +1,152 @@
...
+import TableComponent from "components/Table/Table";
+import moment from "moment";
+import Button from "components/Button/Button";
+import "./TestHistory.scss";
+import { fetchData } from "api/api";
 
-function TestHistory() {
-  return <MaintenancePage />;
+const TestHistory = () => {
+  const [data, setData] = useState<any>([]);
```
Comment (bogdan-manole) [inline react-web/src/pages/testHistory/TestHistory.tsx:9]: why any?

**Review by bogdan-manole — CHANGES_REQUESTED** [review]

File: react-web/src/pages/testHistory/TestHistory.tsx (hunk 2 above)
Comment (amnambiar) [reply react-web/src/pages/testHistory/TestHistory.tsx]: Fixed this

File: react-web/package.json (hunk 1 above)
Comment (amnambiar) [reply react-web/package.json]: Replaced moment with [dayjs](https://day.js.org/docs/en/display/format) as only direct .format was needed

File: react-web/src/pages/testHistory/TestHistory.tsx [hunk 3]
```diff
@@ -5,19 +5,26 @@ import Button from "components/Button/Button";
 import "./TestHistory.scss";
 import { fetchData } from "api/api";
 
+interface ICampaign {
+  "certificateCreatedAt": string,
```
Comment (bogdan-manole) [inline react-web/src/pages/testHistory/TestHistory.tsx:9]: my bad here, this property (`certificateCreatedAt `) doesn't exist anymore in the `ICampaing` record.
Now it belongs to the Certification object retrieved to its separate API route

**Review by bogdan-manole — CHANGES_REQUESTED** [review]

File: react-web/src/pages/testHistory/TestHistory.tsx (hunk 3 above)
Comment (amnambiar) [reply react-web/src/pages/testHistory/TestHistory.tsx]: Removed

**Pull request merged**